src/shared/src/types/change-request-types.ts

```diff
@@ -30,7 +30,7 @@ export const ChangeRequestType = {
   Activation: 'ACTIVATION'
 } as const;
 // eslint-disable-next-line @typescript-eslint/no-redeclare
-export type ChangeRequestType = typeof ChangeRequestType[keyof typeof ChangeRequestType];
+export type ChangeRequestType = (typeof ChangeRequestType)[keyof typeof ChangeRequestType];
```

Comment:
why idid you change this

Reply:
no clue, probably ran prettier on the file by accident